test/T70.module_local.config

```diff
@@ -0,0 +1 @@
++module_local ALL
```

Comment:
Also (optional): since you introduced functionality to select `all` namespace then it seems logical for someone to use this option and then select particular namespace as non-local. Have you consider adding `-module_local_namespace` support?